pom.xml

```diff
@@ -22,14 +22,34 @@
     </license>
   </licenses>
 
+  <properties>
+    <!-- for dynamically installing protoc -->
+    <grpc.version>1.6.0</grpc.version>
```

Comment:
1.6.1? I know that they had a bug in 1.6.0

Reply:
done and ty for the tip